src/sage/combinat/matrices/latin.py

```diff
@@ -793,8 +793,8 @@ def random_empty_cell(self):
 
         OUTPUT:
 
-        - ``[r, c]`` -- cell such that self[r, c] is empty, or returns
-          None if self is a (full) latin square.
+        - ``[r, c]`` -- cell such that ``self[r, c]`` is empty, or returns
+          ``None`` if ``self`` is a (full) latin square
 
```

Comment:
```
Latin square
```

Reply:
It is `latin` in many places. No need to capitalize I think.

Reply:
OK. Out of scope of this PR.